data/categories.txt

```diff
@@ -2344,7 +2344,7 @@ sw:^Makumbusho|kivutio|utalii
 waterway-waterfall|@tourism
 en:2^Waterfall
 ru:2^Водопад|достопримечательность
-ar:^شلال|مناظر
+ar:^شلال|مناظر|سياحة
```

Comment:
Sure, but mixing left-to-right and right-to-left may be confusing.
Apparently you are right here.